onward/app/services/repositories.scala

```diff
@@ -22,10 +21,14 @@ trait Related extends ConciergeRepository {
         case excluding => Some(excluding.map(t => s"-$t").mkString(","))
       }
 
-      val response = getResponse(LiveContentApi.item(path, edition)
+      val requestBuilder = LiveContentApi.item(path, edition)
         .tag(tags)
         .showRelated(true)
-      )
+
+      val response = relatedType match {
+        case Nil => getResponse(requestBuilder)
+        case someType => getResponse(requestBuilder.stringParam("related-type", someType.head))
```

Comment:
here if you are using an Option[String] instead you could case None => and case Some(type) => to avoid having to do a .head later.